routes/chat_routes.py

```
@@ -0,0 +1,380 @@
+    # 新しいメッセージを追加
+    messages.append(HumanMessage(content=message))
+
+    try:
+        ai_message = _get_llm_and_invoke(model_name, messages)
+
+        # 会話履歴の更新
+        add_to_session_history("chat_history", {"human": message, "ai": ai_message})
+
+        return jsonify({"response": SecurityUtils.escape_html(ai_message)})
+    except Exception as e:
+        raise e
```

Ruff (0.14.6)

Remove exception handler; error is immediately re-raised

(TRY203)